docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerJobProperty.java

```diff
@@ -14,6 +15,12 @@
 public class DockerJobProperty extends JobProperty<AbstractProject<?, ?>> {
 
     /**
+     * Image repository name for committed containers (image names)
+     */
+    @DataBoundSetter
+    public String imageRepository;
```

Comment:
[Javadoc](http://stapler.kohsuke.org/apidocs/org/kohsuke/stapler/DataBoundSetter.html) says that it can be used on field 
> The field is discovered through simple reflection, so its name must match the property name, but its access modifier can be anything.

Will it work without setter in case of private field with annotation?